src/core/load-fragment-gperf.gperf.m4

```diff
@@ -208,7 +208,8 @@ $1.IPAddressAllow,               config_parse_ip_address_access,     0,
 $1.IPAddressDeny,                config_parse_ip_address_access,     0,                             offsetof($1, cgroup_context.ip_address_deny)
 $1.IPIngressFilterPath,          config_parse_ip_filter_bpf_progs_ingress, 0,                       offsetof($1, cgroup_context.bpf_cgroup_program_specs)
 $1.IPEgressFilterPath,           config_parse_ip_filter_bpf_progs_egress, 0,                        offsetof($1, cgroup_context.bpf_cgroup_program_specs)
-$1.NetClass,                     config_parse_warn_compat,           DISABLED_LEGACY,               0'
+$1.NetClass,                     config_parse_warn_compat,           DISABLED_LEGACY,               0
+$1.BpfCgroupProgram,             config_parse_bpf_cgroup_program,    0,                             offsetof($1, cgroup_context.bpf_cgroup_program_specs)'
```

Comment:
I think this should just be BPFProgram. I don't think we should expose the relationship to cgroups in the external name. It's mostly an internal concept, that should be mentioned in the docs but not really in the API description I think